issue with kconnect 0.4.0 on ubuntu

**What happened:**
kconnect throwing error on ubuntu.


**What did you expect to happen:**
NO error

**How to reproduce it:**
$ kconnect version
info    kconnect - the Kubernetes Connection Manager CLI        {"version": "0.4.0"}
version: 0.4.0
builddate: "2020-12-21T16:21:00Z"
commithash: 473de0b6fd68af08279b42f6c1306597de64e402
goversion: go1.13.15
platform: linux/amd64
compiler: gc

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xb540a6]

goroutine 1 [running]:
github.com/fidelity/kconnect/internal/commands.reportNewerVersion(0x1442a00, 0xc000102018)
        /home/runner/work/kconnect/kconnect/internal/commands/root.go:272 +0xfe6
github.com/fidelity/kconnect/internal/commands.RootCmd.func3(0xc00049b600, 0x1c26538, 0x0, 0x0, 0x0, 0x0)
        /home/runner/work/kconnect/kconnect/internal/commands/root.go:139 +0x10d
github.com/spf13/cobra.(*Command).execute(0xc00049b600, 0x1c26538, 0x0, 0x0, 0xc00049b600, 0x1c26538)
        /home/runner/go/pkg/mod/github.com/richardcase/cobra@v1.0.1-0.20200717133916-3a09287ba25e/command.go:870 +0x38a
github.com/spf13/cobra.(*Command).ExecuteC(0xc00049a000, 0x0, 0x0, 0x11)
        /home/runner/go/pkg/mod/github.com/richardcase/cobra@v1.0.1-0.20200717133916-3a09287ba25e/command.go:963 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/richardcase/cobra@v1.0.1-0.20200717133916-3a09287ba25e/command.go:900
main.main()
        /home/runner/work/kconnect/kconnect/main.go:47 +0x4aa

**Environment:**
- OS (e.g. from `/etc/os-release`):
- $ cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

--- a/internal/commands/root.go
+++ b/internal/commands/root.go
@@ -269,9 +269,11 @@ func reportNewerVersion() error {
 		}
 	} else {
 		zap.S().Debugw("latest version not retrieved as check interval not exceeded", "diffMins", checkDiff.Minutes(), "savedVersion", cfg.Spec.VersionCheck.LatestReleaseVersion)
-		latestSemver, err = semver.Parse(*cfg.Spec.VersionCheck.LatestReleaseVersion)
-		if err != nil {
-			return fmt.Errorf("parsing saved latest release version %s: %w", *cfg.Spec.VersionCheck.LatestReleaseVersion, err)
+		if cfg.Spec.VersionCheck.LatestReleaseVersion != nil && *cfg.Spec.VersionCheck.LatestReleaseVersion != "" {
+			latestSemver, err = semver.Parse(*cfg.Spec.VersionCheck.LatestReleaseVersion)
+			if err != nil {
+				return fmt.Errorf("parsing saved latest release version %s: %w", *cfg.Spec.VersionCheck.LatestReleaseVersion, err)
+			}
 		}
 	}
 

